api/internal/core/entity/entity.go

```diff
@@ -58,14 +58,6 @@ func (info *BaseInfo) KeyCompat(key string) {
 	}
 }
 
-type BaseInfoSetter interface {
-	GetBaseInfo() *BaseInfo
-}
-
-type BaseInfoGetter interface {
-	GetBaseInfo() *BaseInfo
-}
```

Comment:
oh, please keep them, their responsibilities are different.

Reply:
`BaseInfoGetter` and `BaseInfoSetter` are currently replaced by `GetBaseInfo` interface. Their interface definitions are exactly the same. And, the lines of codes is reduced.

Reply:
ping @nic-chen